app/services/remark_generators/amount_variation_checker.rb

```diff
@@ -28,5 +28,17 @@ def populate_remarks
     def record_type
       @collection.first.class.to_s.underscore.tr('/', '_').to_sym
     end
+
+    def exempt_from_checking
+      childcare_payment? && childcare_disallowed?
+    end
+
+    def childcare_payment?
+      record_type == :outgoings_childcare
+    end
+
+    def childcare_disallowed?
+      @assessment.disposable_income_summary.childcare.zero?
+    end
```

Comment:
Could these methods be extracted and shared between the two generators, either via a base_generator/concern/etc?

Reply:
Excellent point. I had mental note to refactor but got overexcited and opened the PR too early 😁 

I've pushed a commit to change it if you fancy re-reviewing at some point.